src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -108,17 +116,24 @@
                    {:text "Running" :pred #(= "Running" (:status %))}
                    {:text "Exception" :pred #(= "Exception" (:status %))}]
          :columns
-         [{:header [:div {:style {:marginLeft -6}} "Status"] :starting-width 60
-           :content-renderer (fn [data] [StatusCell {:data data}])
-           :filter-by :none}
-          {:header "Workspace" :starting-width 450
-           :content-renderer (fn [data] [WorkspaceCell {:data data}])
-           :filter-by :name}
+         [{:sort-by (fn [m] (:status m))
```

Comment:
You can just do `:sort-by :status`.  But, I don't think we should have sorting on this column anyway.